pytential/qbx/refinement.py

```diff
@@ -310,9 +310,12 @@ def check_expansion_disks_undisturbed_by_sources(self,
 
         from pytential import bind, sym
         center_danger_zone_radii = flatten(
-            bind(stage1_density_discr,
-                sym.expansion_radii(stage1_density_discr.ambient_dim,
-                    granularity=sym.GRANULARITY_CENTER))(self.array_context),
+            bind(
+                stage1_density_discr,
+                sym.interp(None, sym.GRANULARITY_CENTER, sym.ElementwiseMax(
+                    sym.expansion_radii(stage1_density_discr.ambient_dim),
+                    ))
+                )(self.array_context),
             self.array_context)
```

Comment:
With this the following test basically stalls at some point
```
python test_global_qbx.py 'test_source_refinement_3d(_acf, "sphere", partial(mgen.generate_sphere, 1), 4, visualize=True)'
```

Reply:
Questions for debugging:
- Why does that do `INFO:pytential.qbx.refinement:check_expansion_disks_undisturbed_by_sources:` 5 times, when 0 times suffices on `main`?
- Why does it stall?

Reply:
I rebased my local changes in the hopes of fixing those tests.

Kept the branch with that failure around in https://github.com/alexfikl/pytential/tree/node-stretch-factor-failed to play around with.